### ISSUE TITLE
Protobuf 1.6.0 broke the build.

### ISSUE THREAD

**Issue by afck** [SpinResearch/merkle.rs#28]
Protobuf made a backwards-incompatible change without incrementing the major version number, breaking the build.

(A short-term solution would be using `"~1.4.5"` in `Cargo.toml`.)

**Comment by psivesely** [comment]
`cargo test --all-features` and `cargo build --all-features` are both working on my machine using the latest Rust stable `1.26.0 (a77568041 2018-05-07)`. Can you be more specific with what build is broken and provide instructions to reproduce?

**Comment by afck** [comment]
Did you do `cargo update` (or `cargo clean` or use a freshly cloned repository)? Otherwise it will still use the protobuf version in your `Cargo.lock` file. This is the error I get with current `master`:

```
$ cargo test --all-features
   Compiling merkle v1.5.1-pre (file:///home/andreas/git/merkle.rs)
error[E0433]: failed to resolve. Could not find `MessageStatic` in `protobuf`
   --> src/proto/proof.rs:248:21
    |
248 |         ::protobuf::MessageStatic::descriptor_static(None::<Self>)
    |                     ^^^^^^^^^^^^^ Could not find `MessageStatic` in `protobuf`

error[E0433]: failed to resolve. Could not find `MessageStatic` in `protobuf`
   --> src/proto/proof.rs:633:21
    |
633 |         ::protobuf::MessageStatic::descriptor_static(None::<Self>)
    |                     ^^^^^^^^^^^^^ Could not find `MessageStatic` in `protobuf`

error[E0405]: cannot find trait `MessageStatic` in module `protobuf`
   --> src/proto/proof.rs:252:18
    |
252 | impl ::protobuf::MessageStatic for ProofProto {
    |                  ^^^^^^^^^^^^^ not found in `protobuf`

error[E0405]: cannot find trait `MessageStatic` in module `protobuf`
   --> src/proto/proof.rs:637:18
    |
637 | impl ::protobuf::MessageStatic for LemmaProto {
    |                  ^^^^^^^^^^^^^ not found in `protobuf`

error[E0603]: module `core` is private
  --> src/proto/mod.rs:11:5
   |
11 | use protobuf::core::parse_from_bytes;
```

**Comment by psivesely** [comment]
> Did you do cargo update (or cargo clean or use a freshly cloned repository)? Otherwise it will still use the protobuf version in your Cargo.lock file.

Forgot to `update`, thanks. Can now confirm build is failing on my machine with the same errors.

**Comment by romac** [comment]
Thanks for fixing this @nvesely.

@afck I just released version 1.6.0 on crates.io which contains #29.

**Comment by afck** [comment]
Perfect, thank you!
